spec/serve.js

```diff
@@ -65,6 +52,45 @@ describe('production server', function () {
 
   it('should create build files', function () {
     var fileNames = fs.readdirSync(buildDir);
-    assert(fileNames.length = 2);
+    assert.equal(fileNames.length, 3);
+  });
+
+  it('should deliver expected html page', function (done) {
+    fetch('http://localhost:8080/')
+    .then(function (response) {
+      assert(response.ok);
```

Comment:
Yes, you're right - but that's nothing hops does anything about, but rather an express default behavior... I'd postpone for now.